scripts/git-hooks/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+formatFileList="$(git --no-pager diff --name-status --no-color --cached | awk '$1 != "D" && $2 ~ /\.kts|\.kt/ { print $NF}')"
```

Comment:
The original article also covered Java files, but since we will not be developing in Java for this project, we have excluded them.